cluster/gce/gci/configure.sh

```diff
@@ -376,7 +376,16 @@ function try-load-docker-image {
   set +e
   local -r max_attempts=5
   local -i attempt_num=1
-  until timeout 30 ${LOAD_IMAGE_COMMAND:-docker load -i} "${img}"; do
+
+  if [[ "${CONTAINER_RUNTIME_NAME:-}" == "docker" ]]; then
+    load_image_command=${LOAD_IMAGE_COMMAND:-docker load -i}
+  elif [[ "${CONTAINER_RUNTIME_NAME:-}" == "containerd" ]]; then
+    load_image_command=${LOAD_IMAGE_COMMAND:-ctr -n=k8s.io images import}
+  else
+    load_image_command="${LOAD_IMAGE_COMMAND:-}"
+  fi
+
+  until timeout 30 ${load_image_command} "${img}"; do
     if [[ "${attempt_num}" == "${max_attempts}" ]]; then
       echo "Fail to load docker image file ${img} after ${max_attempts} retries. Exit!!"
```

Comment:
Nit:
Can we add  "using ${load_image_command}" after '${max_attempts} retries'?

That is what I struggled when looking at the logs as a new member.

Reply:
@bsdnet Excellent idea! How about we fix it up in a follow up PR? (would you like to file it please?)

Reply:
Sure, I can come out a fix after you commit current fix. 
I want to enable future contributors to avoid my confusion.